### ISSUE TITLE
Provide better color contrast on selected element in the sidenav component 

### ISSUE THREAD

**Issue by bashibozuk** [vmware/clarity#4188]
## The selected element in the left navigation bar is indicated with color with not enough contrast

A grey color is used to visually indicate current selected navigation element. This color in contrast with the white background does not have sufficient non-text contrast.
Certain low vision users will not be able to see the color difference to identify which on is selected.

## How to reproduce

It is very easily reproducible - go to clarity documentation and select a item from the side navigation

## Expected behavior
When a item is selected in the side nav it should be indicated with color with contrast ratio at least 3:1 with the background

## Versions

**App**

* Clarity: 2.3.5

**Device:**

* Type [MacBook]
* OS: [iOS]
* Browser [chrome]
* Version [79]

## Additional notes

N/A

**Comment by bdryanovski** [comment]
I think that this issue is related to [that](https://github.com/vmware/clarity/issues/2677)

**Comment by mathisscott** [comment]
@bashibozuk 
I'm closing this as a duplicate of #2677.
